app/views/layouts/application.html.erb

```diff
@@ -23,10 +23,24 @@
   <body>
   
     <ul>
-      <li> <%= link_to "Home", "/home" %> </li>
-      <li><%= link_to "Create a Post", "/home/new" %> </li>
+      <li> <%= link_to "Home", posts_path() %> </li>
+      <li><%= link_to "Create a Post", new_post_path() %> </li>
```

Comment:
and here and here too

Reply:
Hellloo, i've removed the brackets in both areas :)